Add custom service Set Timer

gjohansson-ST opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Add text on Sensibo Set Timer Service

Added secondary PR to make custom service more user friendly.

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/73072 https://github.com/home-assistant/core/pull/73684
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

frenck committed:
2022.6: Add missing init system breaking change (#22947)

* 2022.6: Add missing init system breaking change

* More cleanups

emontnemery committed:
Improve MQTT siren documentation (#22958)

Co-authored-by: Franck Nijhof <git@frenck.dev>

dependabot[bot] committed:
Bump rouge from 3.28.0 to 3.29.0 (#22940)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump commonmarker from 0.23.4 to 0.23.5 (#22942)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

balloob committed:
2022.6.1

skipishere committed:
Correcting some typos (#22968)

frenck committed:
2022.6: Update release party, fix MQTT example (#22960)

wrt54g committed:
Increase number of integrations. (#22912)

jensihnow committed:
Updates Conrad ESP32-C3 link to international site (#22948)

balloob committed:
2022.6.2

fcfort committed:
Fix spelling on unifi_direct (#22994)

gjohansson-ST committed:
Add custom service Set Timer

wrt54g committed:
Update URLs (#23006)

* Update RaspBeeII URL

* Update URL

* Update URL

* Update URL

* Update URL

* Remove en from phoscon URL

balloob committed:
2022.6.3

wrt54g committed:
Reduce image size (#23000)

patrickdemooij committed:
Broken link to 'SmartMeter Webshop' removed (#23015)

marcusds committed:
Local sync is now enabled by #21491 (#22991)

wrt54g committed:
URL Update (#22998)

mib1185 committed:
Add hint about multiple nic in same subnet for Synology DSM (#23025)

Dimtuhop committed:
Home Assistant URL setting location update (#23023)

frenck commented on source/_integrations/sensibo.markdown:
```diff
@@ -74,6 +74,14 @@ For diagnostics, not automatically displayed on dashboards, these sensors are av
+
+### Set Timer
+
+You can set or delete a timer to Turn on/off your climate device.
```
```suggestion
You can set or delete a timer to turn on/off your climate device.
```

frenck commented on source/_integrations/sensibo.markdown:
```diff
@@ -74,6 +74,14 @@ For diagnostics, not automatically displayed on dashboards, these sensors are av
+
+You can set or delete a timer to Turn on/off your climate device.
+
+Choose entity and minutes and it will automatically turn or or off your device depending on state with the previous configured climate settings.
```
```suggestion
Choose entity and minutes and it will automatically turn on or off your device depending on the state with the previously configured climate settings.
```

bdraco committed:
Move description of how filtering works to an include (#22974)

wrt54g committed:
Remove unnecessary brackets (#22999)

ClementToche committed:
Update alarm_control_panel.mqtt.markdown (#22996)

hall committed:
Add T3950 to Venstar devices (#22989)

dependabot[bot] committed:
Bump actions/setup-node from 3.2.0 to 3.3.0 (#23026)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

fabaff committed:
Remove link to Awesome HA (#23029)

fabaff committed:
Remove HBMQTT and update frontend part (#23031)

wrt54g committed:
More URL Updates (#23032)

balloob committed:
2022.6.4

omerome83 committed:
Update denonavr.markdown (#23034)

wrt54g committed:
More URL Updates (#23035)

agners committed:
Matter Workshop: Mention support of 64-bit versions only (#23038)

MartinHjelmare committed:
Add warning about 700 series controllers (#23041)

Co-authored-by: Franck Nijhof <git@frenck.dev>

JonLaliberte committed:
Fix links in tag docs (#22823)

tkdrob committed:
Add note to Steam regarding friends list privacy (#22841)

allenporter committed:
Update application credentials documentation for remaining integrations (#23044)

emontnemery committed:
Update core upgrade instructions (#23058)

wrt54g committed:
Reduce image size (#23056)

dannytsang committed:
Removed "click on the menu icon" from advanced configuration (#23050)

frenck committed:
Remove broken Z-Wave plus link & text (#23060)

IGastel committed:
Update octoprint.markdown (#22925)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

dennis-bell committed:
improving the documentation for Buienradar condition codes (#23040)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

frenck committed:
Remove incorrect categories from HTTP integration (#23061)

QbaF committed:
Added next models (#23062)

dannytsang committed:
Replaced Configuration with Settings for the onboarding page (#23065)

frenck committed:
Adjust dlink platform example to use a list (#23064)

balloob committed:
Add Matter info

jjlawren committed:
Clarify Plex client compatibility (#23072)

mbo18 committed:
Update links to zone trigger and zone condition (#23070)

balloob committed:
2022.6.5

gjohansson-ST committed:
Merge branch 'next' into sensibo_set_timer

gjohansson-ST committed:
review comments

knwpsk committed:
Update tank_utility.markdown (#23080)

added quotes around the device ID in config.yaml. HA throws errors without the quotes.

cbhiii committed:
Clarified manual polling disable instructions (#23082)

mkmer committed:
Update Aladdin Connect for Config Flow (#23028)

neillt committed:
Update tank_utility.markdown (#23081)

actionpotato committed:
Update daikin.markdown (#23090)

DubhAd committed:
Update pushover.markdown (#23093)

The YAML was broken

balloob committed:
2022.6.6

mmakaay committed:
Add a link to my DSMR project pgae (#23110)

dasdundave committed:
Missing entity_id in automation recipe (#23112)

mosburgerr committed:
Update threshold.markdown (#23113)

balloob committed:
SkyConnect interest

azcn2503 committed:
Update integration.markdown (#23107)

rgriffogoes committed:
Update timer help - minor menu typo/update (#23132)

* Update timer help - minor menu typo/update

* Matching markdown to other helpers documentation

ufodone committed:
Clarify which credentials are needed for the Envisalink configuration (#23097)

* Clarify which credentials are needed.

Added clarification that the username/password required for configuration are for connecting directly to the device and not the credentials for the user's EyezOn account.

* Remove some spaces

Co-authored-by: Fabian Affolter <mail@fabian-affolter.ch>

pavelplus committed:
Update yeelight.markdown (#23131)

Tested Yeelight Arwen Ceiling Light 450S, added to supported models.

wrt54g committed:
Reduce image size (#23134)

* Delete old file

* Reduce image size

* Delete old file

* Reduce image size

* Delete old file

* Reduce image size

* Delete old file

* Reduce image size

* Delete old file

* Reduce image size

* Delete old file

* Reduce image size

* Delete old file

* Reduce image size

* Delete old file

* Reduce image size

* Delete old file

* Reduce image size

* Remove

* Reduce image size (take 2)

gjohansson-ST committed:
Adjust test

gjohansson-ST committed:
Update to new PR

gjohansson-ST committed:
Merge branch 'home-assistant:current' into sensibo_set_timer
